packages/typespec-vscode/src/extension.ts

```diff
@@ -62,6 +63,21 @@ export async function activate(context: ExtensionContext) {
     }),
   );
 
+  context.subscriptions.push(
+    commands.registerCommand(CommandName.InstallImportPackage, async () => {
+      try {
+        const projectFiles = vscode.workspace.workspaceFolders?.[0].uri.fsPath;
```

Comment:
We need to check the corresponding package.json file to help install package instead of always installing to the first workspace folder.